docs/CHANGELOG.md

```diff
@@ -1,123 +1,226 @@
-# Devilution Changelog
-
-### January 15, 2018
-- The concept of Devilution is born
-- Research into Diablo's code and mechanics
-- Research from Jarulf's guide
-
-### February 4, 2018
-- IDA disassembly begin
-- Start adding function names from sanctuary project
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+### June 16, 2019
+- All functions except one are now binery identical to Diablo 1.09b
+
+## 0.9.6
+### May 19, 2019
+- [96% of functions are now binary identical](https://github.com/diasurgical/devilution/milestone/3) to the 1.09b version
+- Fix several item corruption issues introduced in 0.9.0
+
+## 0.9.0
+### May 2, 2019
+- [90% of functions are now binary identical](https://github.com/diasurgical/devilution/milestone/3) to the 1.09b version
+- Mute buttons now work correctly
+
+### April 15, 2019
+- Code is once again compiled as C++ as some parts appear to require despite the indications in RICH
+
+## 0.8.0
+### April 12, 2019
+- [80% of functions are now binary identical](https://github.com/diasurgical/devilution/milestone/3) to the 1.09b version
+- Fixes a few minor issues with generated items
+
+## 0.7.0
+### April 9, 2019
+- [70% of functions are now binary identical](https://github.com/diasurgical/devilution/milestone/3) to the 1.09b version
+- The last of the compiler flags are figgured out
+
+### April 9, 2019
+- The last of the compiler flags are figgured out
```

Comment:
```suggestion
- The last of the compiler flags are figured out
```

Reply:
Hmm, it seems that this is a duplicate sentence, I think we can remove it and the duplicate `April 9, 2019` header.